src/raster/rasterband.jl

```diff
@@ -582,6 +588,32 @@ a valid mask band.
 """
 maskflags(band::AbstractRasterBand) = GDAL.gdalgetmaskflags(band.ptr)
 
+
+"""
+    maskflaginfo(band::AbstractRasterBand)
+
+Returns the flags as in `maskflags`(@ref) but unpacks the bit values into a named
+tuple with the following fields:
+
+* `all_valid`
+* `per_dataset`
+* `alpha`
+* `nodata`
+
+### Returns
+
+A named tuple with unpacked mask flags
+"""
+function maskflaginfo(band::AbstractRasterBand)
+    flags = maskflags(band)
+    (
```

Comment:
minor nit: consider using the `return` keyword explicitly here for consistency with the rest of the file.

Reply:
fixed with https://github.com/yeesian/ArchGDAL.jl/pull/176/commits/3f0bf870edcc8d54901bc1701435b1878b778e79